queries.sql

```diff
@@ -15,3 +15,30 @@ SELECT * FROM animals WHERE neutered = 'true';
 SELECT * FROM animals WHERE animal_name != 'Gabumon';
 

```

Comment:
Please add queries to find answers to the following questions:
- How many animals are there?
- How many animals have never tried to escape?
- What is the average weight of animals?
- Who escapes the most, neutered or not neutered animals?
- What is the minimum and maximum weight of each type of animal?
- What is the average number of escape attempts per animal type of those born between 1990 and 2000?

Reply:
Thank you issues have been resolved